python/spacetime_local/frame.py

```diff
@@ -504,40 +486,43 @@ def __pull(self):
     def __push(self):
         if self.__disconnected:
             return
-        self._instruments['bytes sent'] = 0
-        changes = self.object_store.get_changes()
+        if self.__instrumented:
+            self._instruments['bytes sent'] = 0
+        changes = self.object_store.get_record()
         for host in self.__host_typemap:
-            update_dict = {}
-            for tp in self.__host_typemap[host]["producing"]:
-                if tp.Class() in changes["new"]:
-                    update_dict.setdefault(tp.__realname__, {"new": {}, "mod": {}, "deleted": []})["new"] = changes["new"][tp.Class()]
-            for tp in self.__host_typemap[host]["gettingsetting"]:
-                if tp.Class() in changes["mod"]:
-                    update_dict.setdefault(tp.__realname__, {"new": {}, "mod": {}, "deleted": []})["mod"] = changes["mod"][tp.Class()]
-            for tp in self.__host_typemap[host]["setting"]:
-                if tp.Class() in changes["mod"]:
-                    update_dict.setdefault(tp.__realname__, {"new": {}, "mod": {}, "deleted": []})["mod"] = changes["mod"][tp.Class()]
-            for tp in self.__host_typemap[host]["deleting"]:
-                if tp.Class() in changes["deleted"]:
-                    update_dict.setdefault(tp.__realname__, {"new": {}, "mod": {}, "deleted": []})["deleted"].extend(changes["deleted"][tp.Class()])
-
-            if update_dict:
-                try:
-                    jsonmsg = json.dumps(update_dict)
+            #update_dict = {}
```

Comment:
There is actually a reason. This is required for multi host server scenarios. Something I did not support yet, as I need to splice the dataframe changes according to host. I'll make an issue for it instead and delete these comments.